src/components/message-box/pages/message-box-interactive.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import MessageBox, {COLOR} from '../MessageBox';
+import DocsActiveBlock from 'components/DocsActiveBlock';
+
+const closeCallback = () => undefined;
+
+const MessageBoxes = () => {
```

Comment:
I believe we always used singular nouns as a name

Reply:
actually Box is also in pages as plural

Reply:
![image](https://user-images.githubusercontent.com/39903772/41467352-7dc78444-70a6-11e8-8328-cd9bda55f2dc.png)